cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -746,6 +746,27 @@ Not a syntax change but a semantic one.
 The `EXECUTE BOOSTED` privilege will no longer include an implicit `EXECUTE` privilege when granted.
 That means that to execute a procedure or a function with boosted privileges both `EXECUTE` and `EXECUTE BOOSTED` are needed.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+[GRANT\|DENY] [IMMUTABLE] ...
+----
+a|
+Privileges can be specified as `IMMUTABLE`, which means that they cannot be altered by users with Privilege Management. 
+They can only be administered when running with full access or by disabling auth.
```

Comment:
I'm not sure 'full access' will mean much to most users. Perhaps via the native/core/java/procedure API? I don't know the right word to use here.

Reply:
I suppose for the administrator-type audience the most meaningful way to refer to it might be: 

```suggestion
They can only be administered with auth disabled.
```

Reply:
Yeah, we can keep it brief in the additions / compatibility list. We should probably explain it in detail somewhere though.

Reply:
definitely. this is covered in more detail in the [tutorial](https://github.com/neo-technology/neo4j-manual-modeling/blob/922daf4d82500ddd2a6257bb82c9cec6e942193f/operationsManual/asciidoc/tutorial/tutorial-immutable-privileges.adoc)